src/Row.js

```diff
@@ -40,8 +40,21 @@ export default class Row extends Component {
   }
 
   _panResponder = PanResponder.create({
-    onStartShouldSetPanResponder: () => !this._isDisabled(),
-    onMoveShouldSetPanResponder: () => !this._isDisabled(),
+    onStartShouldSetPanResponder: () => {
+      if (this._isDisabled()) return false;
+
+      return true;
+    },
+
+    onMoveShouldSetPanResponder: () => {
+      if (this._isDisabled()) return false;
+
+      return true;
+    },
```

Comment:
Not required, I just think it reads better than `return !this._isDisabled()`. 

Less mental gymnastics you have to do.